style.css

```diff
@@ -0,0 +1,202 @@
+body {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  font-family: roboto;
+  margin: 0;
+}
+header {
+  height: 72px;
+  flex: 0 0 auto;
+  background-color: #1f2937;
+  padding: 0 20%;
+  font-size: 18px;
+  display: flex;
+}
+
+a {
+  color: #e5e7eb;
+  text-decoration: none;
+}
+
+nav {
+  flex: 1 1 auto;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: left;
+  padding: 10px;
+  gap: 20px;
+}
+
+.heading {
+  margin: 0;
+  margin-right: auto;
+  font-size: 24px;
+  color: #f9faf8;
+}
+
+.main-section {
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 auto;
+  justify-content: space-between;
+  align-content: center;
+}
+
+.hero {
+  height: 300px;
+  display: flex;
+  background-color: #1f2937;
+  flex-direction: column;
+  padding: 0 20%;
+  flex-direction: row;
```

Comment:
Used `flex-direction` twice here. 
Download a stylelint vs code extension to auto check for errors like this.